appveyor.yml

```diff
@@ -10,43 +10,43 @@ environment:
   matrix:
     - PYTHON: "python3"
       CYGWIN: "C:\\cygwin"
-      JAVA_HOME: "C:\\Program Files (x86)\\Java\\jdk1.8.0"
+      JAVA_HOME: "C:\\jdk8"
```

Comment:
does chocolatey take care of the bitness of the host? Or are we now only testing against only one bitness, eg. 32bit?

Reply:
I believe it does.   Looking at the logs cinst was passively installing jdk8 with ant.